src/components/Tokens/TokenDetails/index.tsx

```diff
@@ -293,7 +308,14 @@ export default function TokenDetails({
             />
           )}
         </RightPanel>
-        {!isInfoTDPEnabled && detailedToken && <MobileBalanceSummaryFooter token={detailedToken} />}
+        {detailedToken && (
+          <MobileBalanceSummaryFooter
+            token={detailedToken}
+            pageChainBalance={multiChainBalances?.find(
+              (tokenBalance) => tokenBalance.token?.id === tokenQuery.token?.id
+            )}
```

Comment:
idea: merge `crossChainMap` & `multiChainBalances` together into a map from `chain -> token & balance ` so that you could do:
```suggestion
            pageChainBalance={crossChainMap[chain].balance}
```

if that ends up being overly-complicated to implement you can ignore, but we should def at least simplify down to one naming convention, probably "multiChain"